js/src/components/google-combo-account-card/connected-google-combo-account-card.scss

```diff
@@ -3,4 +3,18 @@
 	.gla-account-card__description {
 		gap: 0;
 	}
+
+	.gla-connected-accounts-actions {
+		color: $black;
+		font-style: normal;
```

Comment:
Likely unnecessary if this component is rendered in the Card body instead of where the helper text is rendered.

Reply:
This is no longer required.